frameworks/helloworld/tests/test_sanity.py

```diff
@@ -173,14 +175,19 @@ def test_pod_info():
     assert task['status']['state'] == 'TASK_RUNNING'
 
 
+@retrying.retry(
+    wait_fixed=10000,
+    stop_max_delay=30000)
+def wait_for_nonempty_properties():
+    """'suppressed' could be missing if the scheduler recently started,
+    loop for a bit just in case
+    """
+    jsonobj = sdk_cmd.svc_cli(config.PACKAGE_NAME, FOLDERED_SERVICE_NAME, 'state properties', json=True)
+    assert len(jsonobj) > 0
```

Comment:
Don't we need a return statement for `retry` to succeed?

Reply:
by default retrying retries all exceptions so using assert is the generally recommended way to trigger a retry unless you specify `retry_on_exception=lambda ex: False`